src/container/acrManager.ts

```diff
@@ -121,6 +122,12 @@ export class AcrManager {
 
             const repoItems: vscode.QuickPickItem[] = [];
             const repos = JSON.parse(catalogResponse).repositories;
+
+            if (!repos) {
+                isEmptyRegistry = true;
+                throw new Error("There is no repository in the registry.");
```

Comment:
`"There is no repository in the registry."`  is this used?

Reply:
yes, it is used in the error message